src/lib.rs

```diff
@@ -15,9 +15,12 @@ pub extern crate gluon_base as base;
 pub extern crate gluon_parser as parser;
 pub extern crate gluon_check as check;
 
+extern crate regex;
```

Comment:
This could be moved in the `regex_bind` module to make it more self contained. Remember to write `use self::regex::etc` to import stuff from it afterwards.